json-schema/schema.json

```diff
@@ -298,6 +314,51 @@
         }
       }
     },
+    "variable": {
+      "title": "Variable Object",
+      "type": "object",
+      "required": [
+        "dimensions"
+      ],
+      "properties": {
+        "description": {
+          "$ref": "#/definitions/description"
+        },
+        "dimensions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
```

Comment:
In theory, we should restrict this to the *keys* of `cube:dimensions`. i.e. each dimension of the variable should be a dimension in the collection / item. Can json-schema express that requirement?

Reply:
Maybe, but that sounds pretty complex to do. I would not know how to do it and I'm probably the one with the most JSON Schema experience in the STAC team. Would not bother with it right now.